AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -43,6 +43,8 @@ arca.live###namuRank
 defacto.com.tr##section[data-name="WebMainRecommendation"]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/76604
 elabcrural.com###simple-sticky-footer-container
+! https://github.com/AdguardTeam/AdguardFilters/issues/65706
+coinlyhub.com##div.highlight
```

Comment:
Change it a bit `coinlyhub.com##div[class^="col-"] > div.breadcrumb-wrap ~ div.highlight`

Reply:
Let's make it safer, otherwise ok.